src/reanimated2/NativeMethods.ts

```diff
@@ -5,19 +5,18 @@ import {
   isWeb,
   shouldBeUseWeb,
 } from './PlatformChecker';
-
-import { Component } from 'react';
-import { RefObjectFunction } from './hook/commonTypes';
+import { AnimatedRef } from './hook/commonTypes';
+import { Component, RefObject } from 'react';
 
 const IS_NATIVE = !shouldBeUseWeb();
 
 export let measure: (
-  animatedRef: RefObjectFunction<Component>
+  animatedRef: RefObject<Component>
```

Comment:
It's like this in current `d.ts`

```TypeScript
  export function measure<T extends Component>(
    animatedRef: RefObject<T>
  ): MeasuredDimensions | null;
```

Can't just change it because of TypeScript errors in our examples - although the issue is likely the improper typing in those examples. I'd like to fix that in following PRs.